diff --git a/node_modules/@jsonforms/core/src/util/type.ts b/node_modules/@jsonforms/core/src/util/type.ts
index 7cdc8e7..c3fca4f 100644
--- a/node_modules/@jsonforms/core/src/util/type.ts
+++ b/node_modules/@jsonforms/core/src/util/type.ts
@@ -152,6 +152,7 @@ export interface Store<S = any, A extends Action = AnyAction> {
    * For more information, see the observable proposal:
    * https://github.com/tc39/proposal-observable
    */
+  // @ts-ignore
   [Symbol.observable](): Observable<S>;
 }
 
@@ -171,6 +172,7 @@ export type Observable<T> = {
    * emission of values from the observable.
    */
   subscribe(observer: Observer<T>): { unsubscribe: Unsubscribe };
+  // @ts-ignore
   [Symbol.observable](): Observable<T>
 };
 
