diff --git a/node_modules/@jsonforms/core/src/util/type.ts b/node_modules/@jsonforms/core/src/util/type.ts
index c017457..cc66181 100644
--- a/node_modules/@jsonforms/core/src/util/type.ts
+++ b/node_modules/@jsonforms/core/src/util/type.ts
@@ -154,6 +154,7 @@ export interface Store<S = any, A extends Action = AnyAction> {
    * For more information, see the observable proposal:
    * https://github.com/tc39/proposal-observable
    */
+  //@ts-ignore
   [Symbol.observable](): Observable<S>;
 }
 
@@ -172,7 +173,10 @@ export type Observable<T> = {
    * be used to unsubscribe the observable from the store, and prevent further
    * emission of values from the observable.
    */
+
   subscribe(observer: Observer<T>): { unsubscribe: Unsubscribe };
+
+  //@ts-ignore
   [Symbol.observable](): Observable<T>;
 };
 
