diff --git a/node_modules/@jsonforms/vue-vanilla/src/util/composition.ts b/node_modules/@jsonforms/vue-vanilla/src/util/composition.ts
index cfb4111..f9f4279 100644
--- a/node_modules/@jsonforms/vue-vanilla/src/util/composition.ts
+++ b/node_modules/@jsonforms/vue-vanilla/src/util/composition.ts
@@ -126,7 +126,7 @@ export const useVanillaArrayControl = <I extends { control: any }>(
       input.control.value.data,
       composePaths(`${index}`, childLabelProp)
     );
-    if (labelValue === undefined || labelValue === null || labelValue === NaN) {
+    if (labelValue === undefined || labelValue === null || labelValue === Number.NaN) {
       return '';
     }
     return `${labelValue}`;
